openpype/modules/ftrack/plugins/publish/integrate_ftrack_note.py

```diff
@@ -45,7 +45,8 @@ def process(self, instance):
         host_name = context.data["hostName"]
         app_name = context.data["appName"]
         app_label = context.data["appLabel"]
-        comment = instance.data["comment"]
+        # context comment is fallback until old Pyblish is removed
+        comment = instance.data["comment"] or context.data.get("comment")
```

Comment:
This is duplicate of what happens here https://github.com/ynput/OpenPype/blob/develop/openpype/plugins/publish/collect_comment.py#L99